### PR TITLE
Remove "Temporary for testing" code

### PR DESCRIPTION
**Pull request by jgfoster**
On the home page is a section titled "Courses with student access". As an instructor I could add and remove a course to which I had instructor access, but it didn't get added to the student access list. I was able to track this down to some debugging code that slipped into production.

### CODE REVIEW

**Comment by jgfoster** [comment]
I have reviewed and hereby sign the CLA

**Comment by jgfoster** [comment]
recheck

**Comment by jgfoster** [comment]
recheck

**Review by nwalters512 — APPROVED** [review]
Excellent find!

File: apps/prairielearn/src/pages/enroll/enroll.html.ts [hunk 1]
```diff
@@ -23,10 +23,6 @@ export function Enroll({
   courseInstances: CourseInstance[];
   resLocals: Record<string, any>;
 }) {
-  // Temporary for testing.
-  courseInstances.forEach((ci) => {
-    ci.instructor_access = false;
-  });
```
Comment (nwalters512) [inline apps/prairielearn/src/pages/enroll/enroll.html.ts:26]: Removing an extraneous blank line. I'd recommend running `make link`/`make format` to catch/fix formatting issues locally: https://prairielearn.readthedocs.io/en/latest/dev-guide/#coding-style

```suggestion
```

**Pull request merged**